conda/recipes/kvikio/meta.yaml

```
@@ -75,7 +75,7 @@ requirements:
     - python
     - numpy >=1.23,<3.0a0
     - cupy >=12.0.0
-    - zarr
+    - zarr <3.0.0,>=2.0.0
```

Typically we use the order lower bound/upper bound, and require an alpha pinning on the upper bound (otherwise alphas/prereleases of 3.x are compatible).
```suggestion
    - zarr >=2.0.0,<3.0.0a0
```